.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.11'
+      - name: Install dependencies
+        run: |
+          python -m pip install pipenv
+          python -m pipenv install -v
+          python -m pipenv shell
```

Comment:
```suggestion
```

Reply:
Was this removed experimentally, maybe we can bring it back? Now pytest should be there